packages/template-retail-react-app/app/components/footer/index.jsx

```diff
@@ -144,14 +144,16 @@ const Footer = ({...otherProps}) => {
                                     variant="filled"
                                     {...styles.localeDropdown}
                                 >
+                                    {/* NOTE: this needs to be a native <option> (and not <chakra.option>)
+                                        It's a workaround to fix hydration error, by making sure that the `selected` property is set */}
                                     {supportedLocaleIds.map((locale) => (
-                                        <LocaleText
-                                            as="option"
-                                            value={locale}
-                                            shortCode={locale}
+                                        <option
                                             key={locale}
-                                            {...styles.localeDropdownOption}
-                                        />
+                                            value={locale}
+                                            style={{color: 'black'}}
```

Comment:
Even if there's no precedent for it and you have to store this in `constant.js` we need a way of "lifting" this so it's not a hardcoded string buried deep in a template. Open to suggestions on where to store, and ideally it would be something chakra-theme related. Let's discuss, but I think this is a blocker for me.